Public/Src/Cache/ContentStore/DistributedTest/ContentLocation/NuCache/RocksDbContentLocationDatabaseTests.cs

```diff
@@ -0,0 +1,337 @@
+
+            DefaultConfiguration = new RocksDbContentLocationDatabaseConfiguration(_workingDirectory.Path / "rocksdb");
+        }
+
+        private async Task RunTest(Action<OperationContext, ContentLocationDatabase> action) => await RunCustomTest(DefaultConfiguration, action);
+
+        private async Task RunCustomTest(ContentLocationDatabaseConfiguration configuration, Action<OperationContext, ContentLocationDatabase> action, OperationContext? overwrite = null)
+        {
+            var tracingContext = new Context(TestGlobal.Logger);
+            var operationContext = overwrite ?? new OperationContext(tracingContext);
+
+            var database = ContentLocationDatabase.Create(Clock, configuration, () => new MachineId[] { });
+            await database.StartupAsync(operationContext).ShouldBeSuccess();
+            database.SetDatabaseMode(isDatabaseWritable: true);
+
+            action(operationContext, database);
+
+            await database.ShutdownAsync(operationContext).ShouldBeSuccess();
+        }
+
+
+        [Fact]
+        public Task GetSelectorsGivesZeroTasks()
+        {
+            var weakFingerprint = Fingerprint.Random();
+
+            return RunTest((context, contentLocationDatabase) =>
+            {
+                IEnumerable<GetSelectorResult> tasks = contentLocationDatabase.GetSelectors(context, weakFingerprint).ToList();
+                Assert.Equal(0, tasks.Count());
+            });
+        }
+
+        [Fact]
+        public Task GetSelectorsGivesSelectors()
+        {
+            var weakFingerprint = Fingerprint.Random();
+            var selector1 = Selector.Random();
+            var selector2 = Selector.Random();
+            var strongFingerprint1 = new StrongFingerprint(weakFingerprint, selector1);
+            var strongFingerprint2 = new StrongFingerprint(weakFingerprint, selector2);
+            var contentHashListWithDeterminism1 = new ContentHashListWithDeterminism(ContentHashList.Random(), CacheDeterminism.None);
+            var contentHashListWithDeterminism2 = new ContentHashListWithDeterminism(ContentHashList.Random(), CacheDeterminism.None);
+
+            return RunTest((context, contentLocationDatabase) =>
+            {
+                contentLocationDatabase.AddOrGetContentHashList(
+                    context, strongFingerprint1, contentHashListWithDeterminism1).ShouldBeSuccess();
+                contentLocationDatabase.AddOrGetContentHashList(
+                    context, strongFingerprint2, contentHashListWithDeterminism2).ShouldBeSuccess();
+
+                List<GetSelectorResult> getSelectorResults = contentLocationDatabase.GetSelectors(context, weakFingerprint).ToList();
+                Assert.Equal(2, getSelectorResults.Count);
+
+                GetSelectorResult r1 = getSelectorResults[0];
+                Assert.True(r1.Succeeded);

```

Comment:
Instead of checking for success explicitely, I encourage using this: `GetSelectorResult r1 = getSelectorResults[0].ShouldBeSuccess()`. This saves space in tests, imo, significantly.